.github/workflows/delegator_generic_slsa3.yml

```diff
@@ -31,16 +31,24 @@ on:
     # Secrets provided as arguments are passed as inputs to the callback Action.
     secrets:
       secret1:
-        description: >
-          Optional secret.
-
-          This argument is passed, unchanged, to the builder's Action's `build-artifacts`.
-        required: false
       secret2:
+      secret3:
+      secret4:
+      secret5:
+      secret6:
+      secret7:
+      secret8:
+      secret9:
+      secret10:
+      secret11:
+      secret12:
+      secret13:
+      secret14:
+      secret15:
         description: >
-          Optional secret.
+          Optional secrets.
 
-          This argument is passed, unchanged, to the builder's Action's `build-artifacts`.
+          The arguments are passed, unchanged, to the builder's Action's `build-artifacts`.
         required: false
```

Comment:
Probably we can just omit the `required` field on `secret15` too. `required` defaults to false anyway.